scripts/install_dependencies/install_openvino_dependencies.sh

```diff
@@ -151,7 +151,7 @@ elif [ "$os" == "ubuntu20.04" ] || [ "$os" == "ubuntu21.10" ] || [ "$os" == "ubu
 
 elif [ "$os" == "rhel8" ] ; then
 
-    pkgs_core=(libtbb2 libpugixml1v5)
+    pkgs_core=(tbb.x86_64 pugixml.x86_64)
     pkgs_opencv_req=(gtk3)
     pkgs_python=(python38 python38-pip)
     pkgs_dev=(
```

Comment:
do we also need to install json-devel for samples? BA requires nlohmann json library, which is supposed to be installed from the system repositories.

Reply:
BTW, does RHEL8 have 64-bits version of zlib-devel package?

Reply:
why do we need x86_64 explicitly? such file can be used for aarch64 RHEL as well..

Reply:
@ilya-lavrenov now the platform requirement is removed